src/settings.ts

```diff
@@ -102,31 +102,56 @@ export class LegendSettings extends FormattingSettingsCompositeCard {
         placeholder: "Title Text"
     });
 
-    public fontFamily = new formattingSettings.FontPicker({
-        name: "fontFamily",
-        displayNameKey: "Visual_Font",
-        value: "Arial"
+    public color = new formattingSettings.ColorPicker({
+        name: "color",
+        displayNameKey: "Visual_Color",
+        descriptionKey: "Visual_Description_Color",
+        value: {value: "black"},
```

Comment:
I would suggest using `#252423` colour as it's used by default in core visuals.

Reply:
changed to #252423